src/components/feedback-widget/buttons.js

```diff
@@ -1,40 +1,187 @@
 import { css } from "@emotion/core"
 import styled from "@emotion/styled"
+import { keyframes } from "@emotion/core"
+import { breakpoints } from "./presets"
+
+const rotation = keyframes`
+  100% {
+    transform: translateX(0.25rem) rotate(360deg);
+  }
+`
+
+export const focusStyle = css`
+  box-shadow: 0 0 0 0.12rem #ffb238;
+  outline: none;
+`
 
 const buttonStyles = css`
+  -webkit-appearance: none;
+  align-items: center;
   background: rebeccapurple;
   border: none;
-  border-radius: 0.5rem;
-  box-shadow: 1px 1px 6px #33333344;
+  border-radius: 2px;
   color: white;
-  font-size: 14px;
+  cursor: pointer;
+  display: flex;
+  font-size: 0.875rem;
   font-family: sans-serif;
-  padding: 0.25rem 0.5rem;
+  padding: 0.3rem 0.75rem;
+  transition: 0.5s;
   z-index: 1;
-  -webkit-appearance: none;
+
+  svg {
+    height: 1.1rem;
+    transform: translateX(0.25rem);
+    width: 1.1rem;
+  }
+
+  &:focus {
+    ${focusStyle}
+  }
 
   &:disabled {
-    opacity: 0.5;
+    cursor: not-allowed;
+    opacity: 0.9;
   }
-`
 
-export const OpenButton = styled("button")`
-  ${buttonStyles};
-  display: block;
-  position: fixed;
-  bottom: 1rem;
-  right: 1rem;
+  &:hover {
+    box-shadow: 0 0 0 0.12rem #ffb23888;
+  }
 `
 
 export const SubmitButton = styled("button")`
   ${buttonStyles};
-  margin-right: 0.5rem;
+
+  .submitting & {
+    svg {
+      animation: ${rotation} 1s linear infinite;
```

Comment:
Can we add prefers-reduced-motion to the animations and transitions so people can turn it off?

Reply:
@marcysutton Sure, but do you mean, for all animations: opening widget, relocating toggle button, this small button icon animation ...?

Reply:
All animations would be ideal...but particularly the part where the widget opens and the close button slides up.

Reply:
@marcysutton 

I was too optimistic when I wrote "sure". :)

I'm on Ubuntu and Android and seems that this feature is not available on these systems. Correct me if I'm wrong. This is the first time I use it.

I added some `@media` queries but I'm not able to test if they work as they should. Could you please check it?

Reply:
Ah yes, reduced-motion will only work on supported platforms (OSX and maybe Windows), in supported browsers (Safari, Firefox and soon Chrome).

Reply:
@marcysutton OK, I turned off all UI animations (when `prefers-reduced-motion: reduce`) and tested it out on iOS11 device. The only left animations are these interaction animations of 'faces'. If you prefer I can turn them off also but IMHO they add some fun to the form and we should not hide them.

Reply:
@marcysutton Any follow-up thoughts here? Or can we merge this?

Reply:
I think go ahead and merge it, and we can make any necessary changes later!